tests/testlib/s2n_testlib.h

```diff
@@ -197,39 +197,44 @@ struct s2n_kem_kat_test_vector {
     const struct s2n_kem *kem;
     const char *kat_file;
     bool (*asm_is_enabled)();
-    S2N_RESULT (*enable_asm)();
-    S2N_RESULT (*disable_asm)();
+    S2N_RESULT(*enable_asm)
+    ();
+    S2N_RESULT(*disable_asm)
+    ();
```

Comment:
This doesn't look right. Does clang-format not like these function pointers because they use `()` instead of `(void)`?

Reply:
No, actually clang-format doesn't know that S2N_RESULT is a type name. Adding that to .clang-format fixed this.